packages/utilities/psammead-locales/moment/ar.js

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable func-names */
+const moment = require('moment');
+require('moment/locale/ar');
+
+// these keys have been overridden to maintain western numerals
+// some other keys have been left as set upstream
+// see https://github.com/moment/moment/blob/develop/src/locale/ar.js
+
+const months = [
+  'يناير/ كانون الثاني',
+  'فبراير/ شباط',
+  'مارس/ آذار',
+  'أبريل/ نيسان',
+  'مايو/ أيار',
+  'يونيو/ حزيران',
+  'يوليو/ تموز',
+  'أغسطس/ آب',
+  'سبتمبر/ أيلول',
+  'أكتوبر/ تشرين الأول',
+  'نوفمبر/ تشرين الثاني',
+  'ديسمبر/ كانون الأول',
+];
+const symbolMap = {
```

Comment:
Are we meant to be mapping numbers to themselves?

Reply:
yes i think we do otherwise eastern arabic numerals will be maintained which is not our desired behaviour

Reply:
```
.replace(/[1234567890]/g, function(match) {
      return numberMap[match];
})
```
Appears to be finding numbers, then replacing them with the value in this Map, but as far as I can tell, its replacing them with themselves e.g `'1': '1',` are the same character

Reply:
Actually we do not need this func, the `postformat` func ctaers for mainting the western numerals